credential-exchange-types/src/format.rs

```diff
@@ -184,8 +184,8 @@ pub struct PasskeyCredential {
     pub user_display_name: String,
     pub user_handle: B64Url,
     pub key: B64Url,
-    #[serde(default, skip_serializing_if = "Vec::is_empty")]
-    pub fido2_extensions: Vec<Fido2Extensions>, // default []
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub fido2_extensions: Option<Vec<Fido2Extensions>>, // default []
```

Comment:
**Issue:** Fido2Extensions [is not an array](https://fidoalliance.org/specs/cx/cxf-v1.0-wd-20241003.html#dict-fido2-extensions) but an optional struct which contains only optional fields

Reply:
🤦, not sure how this happened. I probably got the different extensions confused.